docs/vale.md

```diff
@@ -153,15 +150,21 @@ TokenIgnores = {%.*?%}, \
 *  `{{.*?}}`: Ignores instances where we use `{{ example }}` and the pattern **inside** of the brackets. 
 
 
+## Ignoring text 
+
+Vale can be forced to ignore sections of text using the following syntax:
+<!-- vale off -->
```

Comment:
This doesn't show on the page. Does it need to be inside a code block?

Reply:
Nice catch, yes.